### YamlMime:ManagedReference
items:
- uid: System.DirectoryServices.AccountManagement.PrincipalExistsException
  id: PrincipalExistsException
  children:
  - System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor
  - System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor(System.String)
  - System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: PrincipalExistsException
  nameWithType: PrincipalExistsException
  fullName: System.DirectoryServices.AccountManagement.PrincipalExistsException
  type: Class
  summary: "Által okozott <xref:System.DirectoryServices.AccountManagement.PrincipalCollection.Add>Amikor tett kísérlet, amely már létezik a gyűjteményben, vagy a rendszerbiztonsági tag beszúrása <xref href=&quot;System.DirectoryServices.AccountManagement.Principal.Save&quot;> </xref> mikor történik kísérlet menteni egy új rendszerbiztonsági tag már létezik a tárolóban.</xref:System.DirectoryServices.AccountManagement.PrincipalCollection.Add>"
  syntax:
    content: 'public class PrincipalExistsException : System.DirectoryServices.AccountManagement.PrincipalException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.DirectoryServices.AccountManagement.PrincipalException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor
  id: '#ctor'
  parent: System.DirectoryServices.AccountManagement.PrincipalExistsException
  langs:
  - csharp
  name: PrincipalExistsException()
  nameWithType: PrincipalExistsException.PrincipalExistsException()
  fullName: System.DirectoryServices.AccountManagement.PrincipalExistsException.PrincipalExistsException()
  type: Constructor
  assemblies:
  - System.DirectoryServices.AccountManagement
  namespace: System.DirectoryServices.AccountManagement
  summary: "Egy új példányát példányosítja az <xref href=&quot;System.DirectoryServices.AccountManagement.PrincipalExistsException&quot;> </xref> osztály."
  remarks: "Az alapértelmezett konstruktor <xref:System.DirectoryServices.AccountManagement.PrincipalExistsException>osztály</xref:System.DirectoryServices.AccountManagement.PrincipalExistsException> új példánya A <xref:System.Exception.Message%2A>tulajdonság az inicializáláskor felvesz a rendszer által megadott üzenetet, amely leírja a hibát.</xref:System.Exception.Message%2A> Ez az üzenet figyelembe veszi a rendszer jelenlegi kulturális környezethez. A <xref:System.Exception.InnerException%2A>tulajdonság az inicializáláskor felvesz a `null`.</xref:System.Exception.InnerException%2A>"
  syntax:
    content: public PrincipalExistsException ();
    parameters: []
  overload: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.DirectoryServices.AccountManagement.PrincipalExistsException
  langs:
  - csharp
  name: PrincipalExistsException(String)
  nameWithType: PrincipalExistsException.PrincipalExistsException(String)
  fullName: System.DirectoryServices.AccountManagement.PrincipalExistsException.PrincipalExistsException(String)
  type: Constructor
  assemblies:
  - System.DirectoryServices.AccountManagement
  namespace: System.DirectoryServices.AccountManagement
  summary: "Egy új példányát példányosítja az <xref href=&quot;System.DirectoryServices.AccountManagement.PrincipalExistsException&quot;> </xref> osztály a megadott hiba miatt."
  remarks: "A <xref:System.DirectoryServices.AccountManagement.PrincipalExistsException>példány inicializálva van a <xref:System.Exception.Message%2A>tulajdonság értékének beállítása `message`.</xref:System.Exception.Message%2A> </xref:System.DirectoryServices.AccountManagement.PrincipalExistsException> Ha `message` van `null`, a <xref:System.Exception.Message%2A>tulajdonság az inicializáláskor felvesz a rendszer által megadott üzenetet.</xref:System.Exception.Message%2A>"
  syntax:
    content: public PrincipalExistsException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A hibaüzenet a következő szöveg."
  overload: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.DirectoryServices.AccountManagement.PrincipalExistsException
  langs:
  - csharp
  name: PrincipalExistsException(SerializationInfo,StreamingContext)
  nameWithType: PrincipalExistsException.PrincipalExistsException(SerializationInfo,StreamingContext)
  fullName: System.DirectoryServices.AccountManagement.PrincipalExistsException.PrincipalExistsException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.DirectoryServices.AccountManagement
  namespace: System.DirectoryServices.AccountManagement
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.DirectoryServices.AccountManagement.PrincipalExistsException&quot;> </xref> a megadott osztály <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> és <xref:System.Runtime.Serialization.StreamingContext>példányok.</xref:System.Runtime.Serialization.StreamingContext>"
  syntax:
    content: protected PrincipalExistsException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> , amely tartalmazza a szükséges lehet, mert az új adatokat<xref href=&quot;System.DirectoryServices.AccountManagement.PrincipalExistsException&quot;></xref>"
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A <xref:System.Runtime.Serialization.StreamingContext>, amely tartalmazza a szerializált folyamban, az új társított forrását <xref href=&quot;System.DirectoryServices.AccountManagement.PrincipalExistsException&quot;> </xref> </xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.DirectoryServices.AccountManagement.PrincipalExistsException
  langs:
  - csharp
  name: PrincipalExistsException(String,Exception)
  nameWithType: PrincipalExistsException.PrincipalExistsException(String,Exception)
  fullName: System.DirectoryServices.AccountManagement.PrincipalExistsException.PrincipalExistsException(String,Exception)
  type: Constructor
  assemblies:
  - System.DirectoryServices.AccountManagement
  namespace: System.DirectoryServices.AccountManagement
  summary: "Egy új példányát példányosítja az <xref href=&quot;System.DirectoryServices.AccountManagement.PrincipalExistsException&quot;> </xref> osztály a megadott hibaüzenet és a megadott beágyazott kivétel."
  remarks: "A <xref:System.DirectoryServices.AccountManagement.PrincipalExistsException>példány inicializálva van a <xref:System.Exception.Message%2A>tulajdonság értékének beállítása `message` és a <xref:System.Exception.InnerException%2A>tulajdonság értékének beállítása `innerException`.</xref:System.Exception.InnerException%2A> </xref:System.Exception.Message%2A> </xref:System.DirectoryServices.AccountManagement.PrincipalExistsException> Ha `message` van `null`, a <xref:System.Exception.Message%2A>tulajdonság az inicializáláskor felvesz a rendszer által megadott üzenetet.</xref:System.Exception.Message%2A> A <xref:System.Exception.InnerException%2A>inicializálása `null`.</xref:System.Exception.InnerException%2A>"
  syntax:
    content: public PrincipalExistsException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A hibaüzenet a következő szöveg."
    - id: innerException
      type: System.Exception
      description: "A beágyazott kivételt."
  overload: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.DirectoryServices.AccountManagement.PrincipalException
  isExternal: false
  name: System.DirectoryServices.AccountManagement.PrincipalException
- uid: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor
  parent: System.DirectoryServices.AccountManagement.PrincipalExistsException
  isExternal: false
  name: PrincipalExistsException()
  nameWithType: PrincipalExistsException.PrincipalExistsException()
  fullName: System.DirectoryServices.AccountManagement.PrincipalExistsException.PrincipalExistsException()
- uid: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor(System.String)
  parent: System.DirectoryServices.AccountManagement.PrincipalExistsException
  isExternal: false
  name: PrincipalExistsException(String)
  nameWithType: PrincipalExistsException.PrincipalExistsException(String)
  fullName: System.DirectoryServices.AccountManagement.PrincipalExistsException.PrincipalExistsException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.DirectoryServices.AccountManagement.PrincipalExistsException
  isExternal: false
  name: PrincipalExistsException(SerializationInfo,StreamingContext)
  nameWithType: PrincipalExistsException.PrincipalExistsException(SerializationInfo,StreamingContext)
  fullName: System.DirectoryServices.AccountManagement.PrincipalExistsException.PrincipalExistsException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor(System.String,System.Exception)
  parent: System.DirectoryServices.AccountManagement.PrincipalExistsException
  isExternal: false
  name: PrincipalExistsException(String,Exception)
  nameWithType: PrincipalExistsException.PrincipalExistsException(String,Exception)
  fullName: System.DirectoryServices.AccountManagement.PrincipalExistsException.PrincipalExistsException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.DirectoryServices.AccountManagement.PrincipalExistsException.#ctor*
  parent: System.DirectoryServices.AccountManagement.PrincipalExistsException
  isExternal: false
  name: PrincipalExistsException
  nameWithType: PrincipalExistsException.PrincipalExistsException
